spec/tree-view-spec.coffee

```diff
@@ -3007,6 +3007,70 @@ describe "TreeView", ->
         expect(atom.notifications.getNotifications()[0].getMessage()).toContain 'Opening folder in Finder failed'
         expect(atom.notifications.getNotifications()[0].getDetail()).toContain 'ENOENT'
 
+  describe "showCurrentFileInFileManager()", ->
+    it "does nothing when no file is opened", ->
+      expect(atom.workspace.getPaneItems().length).toBe(0)
+      expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
```

Comment:
Currently it returns a `notification` :

`atom.notifications.addError("notifyMsg", {...})`
(line488 `tree-view.coffee`)

I'm gonna write a spec of _successfully opening_
But I should kill the _file manger process_ at the end.

So, I think it's a great idea to make `showCurrentFileInFileManager()` returns
_file manger process_ to be manipulated in spec.